CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## vUNREASED
+
+#### New features and UX changes
+
+- Add pod creation and start times to `rkt list` and `rkt status` ([#2030](https://github.com/coreos/rkt/pull/2030)).
```

Comment:
Add a link to [list.md](https://github.com/coreos/rkt/blob/master/Documentation/subcommands/list.md) as well?